api/src/utils.js

```diff
@@ -11,3 +13,9 @@ export function camelToSnakeCaseKeys (obj = {}) {
     return o
   }, {})
 }
+
+export function composeSignedMessage (daoAddress, { name, description, links, icon }) {
```

Comment:
Add a docstring or possibly rename to make it more clear that this is not a generic function to compose any signed message? Or possibly make it a bit more generic? Either one is fine 😊

Reply:
Done in https://github.com/1Hive/apiary/pull/181/commits/9c7117bb63f9dc5bd0ac5964dc70176a621da71b.